layersvt/VkLayer_device_simulation.json.in

```diff
@@ -21,8 +21,48 @@
         "features": {
             "presets": [
                 {
-                    "label": "macOS portability",
-                    "description": "Check the application for Vulkan portability subset.",
+                    "label": "Desktop Vulkan 1.0 Portability",
```

Comment:
The preset is labeled as "Portability", but the config file it uses is called "common_desktop_1_0.json".

Is this common portability minimums, or common desktop minimums?

Reply:
True I should make it more consistent. 

It's not the minimums, it's effectively what's available in the ecosystem.

Reply:
So if I'm understanding the name changes correctly, these are config files with features and limits common between fully conformant desktop implementations, and NOT portability subsets?

Reply:
Yes it's "fully conformant desktop implementations" but that include portability subset as it's required for cross platform support (macOS).